### PR TITLE
Move master to 0.7-dev

### LINKED ISSUE #460: Move master to 0.7.0-dev

### PR DESCRIPTION
**Pull request by seanpmorgan**
Closes #460

### CODE REVIEW

File: scenario_configs/resisc45_baseline_densenet121_finetune.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
         "name": "ImageClassificationTask"
     },
     "sysconfig": {
-        "docker_image": "twosixarmory/tf1:0.6.0-dev",
+        "docker_image": "twosixarmory/tf1:0.7.0-dev-dev",
```
Comment (davidslater) [inline scenario_configs/resisc45_baseline_densenet121_finetune.json:41]: -dev-dev ?

**Comment by davidslater** [comment]
What about adding a test that ensures that the "docker_image" field in each of the configs in `scenario_configs` matches the current version of armory?

File: scenario_configs/resisc45_baseline_densenet121_finetune.json (hunk 1 above)
Comment (seanpmorgan) [reply scenario_configs/resisc45_baseline_densenet121_finetune.json]: Fixed thx.

**Comment by seanpmorgan** [comment]
> What about adding a test that ensures that the "docker_image" field in each of the configs in `scenario_configs` matches the current version of armory?

Good idea. Added

**Pull request merged**